src/arr.ts

```diff
@@ -37,6 +46,61 @@ export class Arr<T> {
     return this.filter(Boolean)
   }
 
+  /**
+   * Breaks the array into multiple, smaller arrays
+   * of the given `size`.
+   *
+   * @param {Number} size
+   *
+   * @returns {Array}
+   */
+  chunk (size: number): any[] {

```

Comment:
I would like to improve the return type here. We can use `T[][]` instead of `any[]`. Do you agree?

```suggestion
  chunk (size: number): T[][] {

```